internal_cert_management/tests/test-env-loading.sh

```
@@ -0,0 +1,172 @@
+)
+
+echo ""
+echo "=== Testing --force Flag Logic ==="
+echo ""
+
+# Test 4: --force flag sets FORCE_SKIP_VALIDATION=true
+echo "Test 4: --force flag parsing"
+(
+    FORCE_SKIP_VALIDATION=false
+    # Simulate argument parsing
+    if [[ "--force" == "--force" ]]; then
```

Shellcheck (0.11.0)

[warning] This expression is constant. Did you forget the $ on a variable?

(SC2050)